strapi/strapi
# CS-115: NPS Banner UI component

### What does it do?

NPS Banner UI component added with dumb submit button.

### Why is it needed?

[Issue](https://strapi-inc.atlassian.net/browse/CS-115)

### How to test it?

If the user checked the “Keep me updated” box when registering to Strapi, then user will see the NPS survey banner with the form to fill out at the bottom of the screen.
